cobbler/tftpgen.py

```diff
@@ -80,40 +80,41 @@ def copy_images(self):
                 errors.append(e)
                 self.logger.error(e.value)
 
-        # FIXME: using logging module so this ends up in cobbler.log?
+    def copy_single_distro_file(self, d_file, distro_dir, symlink_ok):
+        """
+        Copy a single file (kernel/initrd) to distro's images directory
+
+        :param: d_file:     distro's kernel/initrd value
+        :type:  d_file:     str
+        :param: distro_dir: directory (typically in {www,tftp}/images) where to copy the file
+        :type:  distro_dir: str
+        :param: symlink_ok: whethere it is ok to symlink the file. Typically false in case the file
+                            is used by daemons run in chroot environments (tftpd,..)
+        :type:  symlink_ok: bool
+        :returns: None
```

Comment:
Still the wrong format: https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html